src/docs/product/integrations/integration-platform/ui-components/formfield.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: FormField Components
+sidebar_order: 4
+description: "Learn about FormField components."
+---
+
+The issue linking and alert rule action UI components can utilize `FormField` components to display attributes in their respective modals. A `FormField` can be one of three types (which behave like their HTML equivalents) — `Text`, `TextArea,` and `Select`.
```

Comment:
I wouldn't mention specific UI components here in case we add more that accept FormFields. Maybe only as examples:

> Some UI Components (e.g. Issue Linking, Alert Rule Action) can render custom forms with inputs specified according to the schemes below. They behave similar to their HTML equivalents.

Reply:
I also wouldn't list out Text, TextArea and Select, since it'd be on someone to remember or notice they must update this sentence if we add another field type later on.